### ISSUE TITLE
How to upgrade to the version 3.0.0 with brew

### ISSUE THREAD

**Issue by kissu** [thoughtbot/parity#159]
Hi, I'm not able to upgrade to the version 3.0.0 with Brew. 🍺 

I ran `brew update` twice, `brew tap thoughtbot/formulae --full` and there is no version `3.0.0` in it.
I tried `brew outdated` and as expected, there is no presence of `parity`.
I also tried `brew upgrade parity@3.0.0` but there is no available formulae for this one.

Did I miss something ? Like is it still in beta ?

## Some information about my installation

- MacOS 10.13.5
- I got `/usr/local/bin/development`, `/usr/local/bin/staging` and `/usr/local/bin/production` respectively.
- `brew list` gives me
```
/usr/local/Cellar/parity/2.4.0/bin/development
/usr/local/Cellar/parity/2.4.0/bin/production
/usr/local/Cellar/parity/2.4.0/bin/staging
/usr/local/Cellar/parity/2.4.0/lib/parity/ (5 files)
/usr/local/Cellar/parity/2.4.0/lib/parity.rb
```

**Comment by geoffharcourt** [comment]
Hi @kissu, we need this PR to be merged in order to have Parity 3.0.0 via Homebrew: https://github.com/thoughtbot/homebrew-formulae/pulls/51

**Comment by kissu** [comment]
Thank you for the clarification and for pinging the dudes responsible of the repo. 🙏

**Comment by nickcharlton** [comment]
Closing as https://github.com/thoughtbot/homebrew-formulae/pull/51 was also merged.

**Comment by kissu** [comment]
![image](https://user-images.githubusercontent.com/5133074/47348376-a32d2380-d6b1-11e8-8f62-e592603df5f0.png)

Apparently I'm in v3.0.0 today but I still cannot use `pr_app`, did I miss something ? 😅

**Comment by geoffharcourt** [comment]
The formula didn't add the new `pr_app` executable. This worked in my testing because I had the gem version already installed. I put up https://github.com/thoughtbot/homebrew-formulae/pull/55 to resolve this.

**Comment by mjankowski** [comment]
I just merged the `3.0.0.1` homebrew formula release, which does seem to resolve the issue of no pr_app being included.

**Comment by kissu** [comment]
Thanks all, it's working. 🎉
